ext/test/zpages/tracez_data_aggregator_tests.cc

```diff
@@ -0,0 +1,531 @@
+#include "opentelemetry/sdk/trace/recordable.h"
```

Comment:
Nice tests! 
Can you also add a tests that runs GetAggregatedData twice and checks that the content is the same after the second call as well? (Without any changes in running/completed in between).